timelock-impl/src/main/java/com/palantir/atlasdb/timelock/auth/impl/SimpleAuthorizer.java

```diff
@@ -23,34 +23,34 @@
 import com.palantir.atlasdb.timelock.auth.api.Privileges;
 import com.palantir.lock.TimelockNamespace;
 import com.palantir.atlasdb.timelock.auth.api.Authorizer;
-import com.palantir.atlasdb.timelock.auth.api.Client;
+import com.palantir.atlasdb.timelock.auth.api.AuthenticatedClient;
 
 public class SimpleAuthorizer implements Authorizer {
-    private final Map<Client, Privileges> privileges;
+    private final Map<AuthenticatedClient, Privileges> privileges;
     private final NamespaceLocker namespaceLocker;
 
     @VisibleForTesting
-    SimpleAuthorizer(Map<Client, Privileges> privileges, NamespaceLocker namespaceLocker) {
+    SimpleAuthorizer(Map<AuthenticatedClient, Privileges> privileges, NamespaceLocker namespaceLocker) {
         this.privileges = privileges;
         this.namespaceLocker = namespaceLocker;
     }
 
-    public static Authorizer of(Map<Client, Privileges> privileges, AuthRequirement authRequirement) {
+    public static Authorizer of(Map<AuthenticatedClient, Privileges> privileges, AuthRequirement authRequirement) {
         NamespaceLocker namespaceLocker = createNamespaceLocker(authRequirement, privileges);
         return new SimpleAuthorizer(
                 ImmutableMap.copyOf(privileges),
                 namespaceLocker);
     }
 
     @Override
-    public boolean isAuthorized(Client client, TimelockNamespace namespace) {
+    public boolean isAuthorized(AuthenticatedClient authenticatedClient, TimelockNamespace namespace) {
```

Comment:
let `AuthenticatedClient` contain `ClientId`, and then you can do `authenticatedClient.id()`